pyramid/scripts/pshell.py

```diff
@@ -159,18 +162,33 @@ def run(self, shell=None):
             closer()
 
     def make_shell(self):
+        shells = {}
+
+        priority_order = ['ipython', 'bpython']
```

Comment:
I'm not sure there's any real reason why we need to keep these at a higher priority than other entry points.